Trimble.Modus.Components/Styles/Colors.xaml

```diff
@@ -51,4 +51,7 @@
   <!-- Switch -->
   <Color x:Key="SwitchSelected">#217CBB</Color>
   <Color x:Key="SwitchUnselected">#90939f</Color>
-</ResourceDictionary>
\ No newline at end of file
+  <!--DropDown-->
+  <Color x:Key="DropDownListBorder">#217CBB</Color>
```

Comment:
SwitchSelected color and DropDownListBorder having the same color code. Please refactor this

Reply:
The refactoring of color codes will be done after pushing the remaining components including this.

Reply:
Yeah, there's a whole lot changes need to be done for the Colors. We have talked about this today. Let's cover this as part of that ticket